src/Features/Lsif/Generator/ResultSetTracking/DelegatingResultSetTracker.cs

```diff
@@ -2,8 +2,6 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-#nullable disable
```

Comment:
Does this mean this in now enabled and we should review the annotations in the file?

Reply:
Before the default flip these were already enabled via a project level setting and had been reviewed. Jason caught that in my PR to flip the defaults. Rather than reset that PR though he advised me to move forward with the default flip and send a second PR to get us back to the prior state. This is that PR. 

As a part of that we went through the places where all the project level settings had been changed and reset all of the files.